lib/matplotlib/tests/test_patches.py

```diff
@@ -677,8 +677,7 @@ def test_contains_points():
     assert np.all(result == expected)
 
 
-# Currently fails with pdf/svg, probably because some parts assume a dpi of 72.
```

Comment:
Should we leave this comment?

Reply:
It looks like it does still fail on PDF/SVG, so we can leave it in.